migrator.go

```diff
@@ -0,0 +1,33 @@
+package pspmigrator
+
+import (
+	v1 "k8s.io/api/core/v1"
+
+	"k8s.io/pod-security-admission/api"
+	psaApi "k8s.io/pod-security-admission/api"
+	"k8s.io/pod-security-admission/policy"
+)
+
+func SuggestedPodSecurityStandard(pod *v1.Pod) (psaApi.Level, error) {
+	evaluator, err := policy.NewEvaluator(policy.DefaultChecks())
+	if err != nil {
+		return "", err
+	}
+	apiVersion, err := api.ParseVersion("latest")
+	if err != nil {
+		return "", err
+	}
```

Comment:
```suggestion
	apiVersion := api.LatestVersion()
```

or just inline that below.

Reply:
done